### ISSUE TITLE
AWS Certificate documentation for our PR preview documentation

### ISSUE THREAD

**Issue by chachasikes** [cagov/odi-engineering#86]
Our wiki page for PR previews mentions "managing AWS certificates"

We are getting the PR preview set up for upcoming cannabis headless work 
https://github.com/cagov/Help-Desk/issues/652

Aaron needed to do this step, as according this this documentation (which is very good - seems like it's just a few notes on the final part of the process that aren't entirely self-service.)

https://github.com/cagov/odi-engineering/wiki/Set-up-PR-preview-sites

I'm attaching a screenshot
![Screen Shot 2022-08-12 at 2 30 03 PM](https://user-images.githubusercontent.com/149436/184447741-e3f501b7-24ff-42b5-9547-c2b7f92261ea.png)

Seems like AWS has a documentation labyrinth - or I'm missing some permissions.
https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/welcome

* I don't have the UI mentioned in "Otherwise, choose Certificate Manager or Private CAs in the left navigation pane."
* Nor do I have a Get Started button.

I wouldn't have been able to complete this & would need to spend a lot of time with the documentation - or call for support.

Note: This is super common with AWS documentation. 
They often have one route that's documented & a legacy app pointing somewhere else - so it's hard to learn unless you have corporate support. Usually there is a training that happens - and then the person who attended the training will give live demos to team mates. (I think it was Chad who mentioned he had this training in the past.)

For working with AWS documentation (and most other corporate created documentation) - it's can be helpful to:
* include screenshots (with obfuscated data, sensitive info, if any etc)
* include URL patterns to the page you actually are visiting - which can help
* record a Loom video or video (and host it in Google Drive) and link to the documentation.

**Comment by xjensen** [comment]
Regarding the screenshot, is that all you see? The page looks scrolled down. 

This is what I get.

<img width="1878" alt="Screen Shot 2022-08-12 at 15 40 54" src="https://user-images.githubusercontent.com/1208960/184454762-b9617eed-df9a-4ff3-a91c-aaf9b836481e.png">

**Comment by chachasikes** [comment]
Thanks Jon! That's helpful - I see the menu hamburger today (which I did not see on Friday)

It's very subtle & is also different from how AWS has done menus that I have used in the past.

Suggestions for docs:
* Look for a menu hamburger on the left
* Expand it
* Choose "List Certificates"

Attaching more screenshots of how it looks when the menu is expanded.

![Screen Shot 2022-08-15 at 9 26 34 AM](https://user-images.githubusercontent.com/149436/184675669-c3c9d611-cc7b-46fa-af0d-8a9b8f33c4dc.png)

![Screen Shot 2022-08-15 at 9 26 39 AM](https://user-images.githubusercontent.com/149436/184675666-741672d0-280a-4ff1-8e95-cfdc2682ce1f.png)
